app/views/users/edit.html

```diff
@@ -0,0 +1,102 @@
+<h2>{{action}} User</h2>
```

Comment:
I plan to re-use this same html file for the new user. Therefore I want to bind the action name, which will be either **Edit** or **New**.